httpclient5-cache/src/main/java/org/apache/hc/client5/http/impl/cache/CachingExec.java

```diff
@@ -290,38 +293,38 @@ private ClassicHttpResponse handleCacheHit(
 
         if (requestContainsNoCacheDirective(request)) {
```

Comment:
We can The method 'requestContainsNoCacheDirective' also parses the request, but we have already done that. We should simply pass the RequestCacheControl. Also, it should be RequestCacheControl, not ResponseCacheControl.

Reply:
@ok2c Corrected.